client/models.py

```diff
@@ -281,7 +284,62 @@ def image_download_url(self):
             return reverse('download_certificate', kwargs={'pk': self.id})
 
 
+class HelpRequestVolunteerManager(models.Manager):
+    def _coord_distance(self, p1, p2):
```

Comment:
It seems like there's no current usage for this function as you perform the calculations on the DB (in _add_distance)